app/templates/admin/users/users.html

```diff
@@ -60,6 +51,59 @@
             </a>
         </div>
     </div>
+    <div class="display-feature-table">
```

Comment:
You should use jinja to generate this!

Here are the steps:

- Define a data structure holding the data:

```html
{% set features = [
  ('Author blog posts',          [True, True, True, True]),
  ('Create and edit blog posts', [False, True, True, True]),
  ...
] %}
```

- Use jinja to generate it:

```html
{% for description, bools in features %}
<tr>
  <th>...</th>
  {% for bool in bools %}
  <td>
    {% if bool %}
    ...
    {% else %}
    ...
    {% endif %}
  {% endfor %}
</tr>
{% endfor %}
```